image/tarexport/load.go

```diff
@@ -125,7 +131,15 @@ func (l *tarexporter) loadLayer(filename string, rootFS image.RootFS) (layer.Lay
 	}
 	defer inflatedLayerData.Close()
 
-	return l.ls.Register(inflatedLayerData, rootFS.ChainID())
+	fileInfo, err := os.Stat(filename)
+	if err != nil {
+		logrus.Debugf("Error stat file: %v", err)
+		return nil, err
+	}
+
+	progressReader := progress.NewProgressReader(inflatedLayerData, progressOutput, fileInfo.Size(), "", fmt.Sprintf("Loading layer: %s", stringid.TruncateID(id)))
```

Comment:
I think it would be more consistent with push/pull to do something like this:

```go
progressReader := progress.NewProgressReader(inflatedLayerData, progressOutput, fileInfo.Size(), stringid.TruncateID(id), "Loading layer")
```